chokuretsu/guide/dump-rom.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
+    <meta name="description"
+          content="English patch for Nintendo DS game The Series of Haruhi Suzumiya (Suzumiya Haruhi no Chokuretsu)"/>
+    <title>The Series of Haruhi Suzumiya - Setup Guide</title>
+    <link rel="stylesheet" href="../style.css">
+
+    <!-- Loads the footer and sidebar -->
+    <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+    <script>
+        $(function () {
+            $("footer").load("../footer.html");
+            $(".column-with-sidebar-bar").load("sidebar.html");
+        });
+    </script>
+
+    <!-- Web icons -->
+    <link rel="manifest" href="../manifest.json"/>
+    <link rel="shortcut icon" href="../assets/icons/favicon.ico" type="image/png" sizes="16x16"/>
+    <link rel="shortcut icon" href="../assets/icons/android-chrome-192x192.png" type="image/png" sizes="192x192"/>
+
+    <!-- Google fonts (for Nunito font) -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito">
+
+    <!-- Font awesome (for icons) -->
+    <script src="https://kit.fontawesome.com/aa4c023dda.js" crossorigin="anonymous"></script>
+
+    <!-- Platform detection script -->
+    <script src="guide.js" type="text/javascript"></script>
+</head>
+<body>
+<main id="column-with-sidebar">
+    <meta id="redirect-if-device-not-selected">
+    <div class="column-with-sidebar-body">
+        <div class="one-column-text" id="game-logo-container">
+            <object class="image-graphic drop-shadow" width="100%" data="../assets/game-logo.svg" type="image/svg+xml">
+                <img src="../assets/game-logo.png" alt="The Series of Haruhi Suzumiya logo"/>
+            </object>
+        </div>
+
+        <a href="choose-platform.html">
+            <div class="guide-platform-indicator hidden drop-shadow">
+                <object id="guide-platform-indicator-icon" width="20em" type="image/svg+xml"></object>
+                <b id="guide-platform-indicator-text"></b>
+            </div>
+        </a>
+
+        <div class="one-column-text">
+            <h1>Dumping the ROM</h1>
+            <p>You'll now need to follow the steps to extract the ROM file from your game cartridge.</p>
```

Comment:
I feel like either now or earlier we should include some information to the effect of "Have multiple devices? The easiest to dump from is the 3DS, followed by the DSi, followed by the DS" or some sort of opinion there to help people choose which path to take.

Reply:
I've added difficulty ratings to each one and added information at the top of the choose device page about this.